wordpress-mobile/WordPress-iOS
# Deprecating WPTableViewController (YAY!)

#### Details:
In this PR we'll nuke **WPTableViewController**, and update the last ViewController that was still depending on it, to rely on **WPTableViewHandler** + **WPContentSyncHelper** instead.

This is based on the work already done by Eric in `ReaderCommentsViewController`.

--

#### Test Scenario 1: Infinite Scroll
1. Launch WPiOS
2. Tap over **My Sites**
3. Pick any of your blogs
4. Tap over **Comments**
5. Scroll all the way down

*Expected:* a spinner shows up at the bottom, and the app begins loading more comments

--

#### Test Scenario 2: Empty State
1. Launch WPiOS
2. Tap over **My Sites**
3. Pick any of your blogs that *does not have any comments*
4. Tap over **Comments**

*Expected:* A label shows up onscreen, indicating that there are no comments to be displayed.

--

#### Test Scenario 3: Pull to Refresh
1. Launch WPiOS
2. Tap over **My Sites**
3. Pick any of your blogs
4. Tap over **Comments**
5. On the web, add a new comment
6. Perform the *Pull to Refresh* gesture

*Expected:* A spinner shows up onscreen, and after a couple seconds, the new comment shows up onscreen.

@aerych can i poke you with this one?. I think you'd love the **WPTableViewController** part!

Thank you!
